plugins/destination/mssql/client/unicode.go

```diff
@@ -0,0 +1,25 @@
+package client
+
+import (
+	"strconv"
+)
+
+func unescape(str string) string {
```

Comment:
Can we add some test cases for this logic? That would make it easier to understand what problem this code is solving

Reply:
done in 6e83ed0